src/components/homePage/Faq.vue

```diff
@@ -1,19 +1,68 @@
 <template>
-<div id="faq">
-	<h2>Frequently Asked Questions</h2>
-	<!-- CREATE A DROPDOWN COMPONENT -->
-</div>
+  <div id="faq">
+    <h2>Frequently Asked Questions</h2>
+    <div class="questions" v-for="question in questions['questions']" :key="question">
+      <button @click="show" class="accordion">{{question.question}}</button>
+      <div class="content">
+        <p>{{question.answer}}</p>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+import Questions from "../../content/faq.json";
 export default {
-	name: "Faq",
-}
+  name: "Faq",
+  data() {
+    return {
+      questions: Questions,
+      lastAccordion: ""
+    };
+  },
+  methods: {
+    show: function(event) {
```

Comment:
Would it be possible to have the different CSS for "visible" vs "hidden" defined in CSS classes, and the show method just have toggle them? That'll make the code easier to extend if design's layout ends up being more complex.

Reply:
Oh, I see Daniel already suggested moving to a pure-Vue implementation in the future. That's cool, we can do that then. :)

Reply:
@namansood That's kind of what I was thinking actually, but I think it's okay for now. I'm good with the current implementation if you are.